common/src/main/resources/assets/wynntils/urls.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "version": 5
```

Comment:
Can you make a PR for API repo?

Reply:
Sure, thanks for reminding me! https://github.com/Wynntils/WynntilsWebsite-API/pull/46